.travis.yml

```diff
@@ -0,0 +1,24 @@
+
+dist: bionic
+sudo: false
+
+if: type = push
```

Comment:
Do you need this? Our other travis files don't have it so I'm just curious what it's doing.

Reply:
It depends on how we have Travis setup in the UI. If we have listening on pull_request & push events we'll get duplicate builds if a PR is opened, so this limits that to only running on a push event instead of both.

Reply:
sounds good - removed